grommet/grommet
# Radio button docs

What does this PR do?

Doc improvements to Radio Button
Where should the reviewer start?
/src/radioButton/docs
src/js/RangeSelector/doc.js
What testing has been done on this PR?
How should this be manually tested?
Any background context you want to provide?
What are the relevant issues?

#2818
Screenshots (if appropriate)
Do the grommet docs need to be updated?
auto
Should this PR be mentioned in the release notes?
no
Is this change backwards compatible or is it a breaking change?
backward compatible
